.dockerignore

```diff
@@ -2,3 +2,5 @@
 !assembly/assembly-main/target/eclipse-che-*
 assembly/assembly-main/target/eclipse-che-*.tar.gz
 assembly/assembly-main/target/eclipse-che-*.zip
+
+!assembly/assembly-main/src/assembly/bin/launcher.sh
```

Comment:
Launcher.sh is in `dockerfiles/che/` not in `assembly/assembly-main/src/assembly/bin/`. And by the way do you really want to add `launcher.sh` in the build context of the che-server?